python/tvm/relay/frontend/pytorch.py

```diff
@@ -3844,6 +3844,30 @@ def inplace_copy(self, inputs, input_types):
         # Return
         return _op.scatter_nd(source, indices, values, mode)
 
+    def linalg_vector_norm(self, inputs, input_types):
+        data = inputs[0]
```

Comment:
It looks like this method is based on torch.linalg.vector_norm. The latter assums that input data type is float, double or complex one; dtype also should be real or complex. Would you check it?

Reply:
Thanks for your review. That's true. It's based on torch.linalg.vector_norm and it supports complex input data. Seems like the PyTorch frontend doesn't support complex data type. [_convert_data_type](https://github.com/mshr-h/tvm/blob/f2e6107b56bb921306f4969ff03251e13686e4c1/python/tvm/relay/frontend/pytorch.py#L4512)
Should I add complex data type support?

Reply:
Hello @mshr-h! My idea was to add something like `assert data.dtype == float or data.dtype == double`. And may be add TODO for further support complex values, but I do not think it is needed just now

Reply:
Hi @vvchernov ! I've added assertion and testcases for double-precision input data.